### ISSUE TITLE
Downloading manifest fails, code 401

### ISSUE THREAD

**Issue by Reimnop** [SteamRE/DepotDownloader#523]
### What did you expect to happen?

It's supposed to download successfully.

### Instead of that, what actually happened?

It fails with the error: "Encountered 401 for depot manifest 440312 5480926750400450243. Aborting."

### Which operating system are you running on?

Windows

### Version

2.6.0.0

### Relevant log output

```shell
Using Steam3 suggested CellID: 71
Got 68 licenses for account!
Got AppInfo for 440310
Using app branch: 'public'.
Got depot key for 440312 result: OK
Processing depot 440312
Downloading depot manifest...Got manifest request code for 440312 5480926750400450243 result: 0
No manifest request code was returned for 440312 5480926750400450243
[ContentDownloader] Downloading manifest 5480926750400450243 from cache4-hkg1.steamcontent.com:443 (SteamCache) with no proxy
14:52:03.4518013  System.Net.Http.RequestStart(scheme: https, host: cache4-hkg1.steamcontent.com, port: 443, pathAndQuery: /depot/440312/manifest/5480926750400450243/5, versionMajor: 1, versionMinor: 1, versionPolicy: 0)
14:52:03.4525436  System.Net.NameResolution.ResolutionStart(hostNameOrAddress: cache4-hkg1.steamcontent.com)
14:52:03.4528916  System.Net.NameResolution.ResolutionStop()
14:52:03.4530467  System.Net.Sockets.ConnectStart(address: InterNetwork:16:{1,187,103,28,54,132,0,0,0,0,0,0,0,0})
14:52:03.4805216  System.Net.Sockets.ConnectStop()
14:52:03.4807177  System.Net.Security.HandshakeStart(isServer: False, targetHost: cache4-hkg1.steamcontent.com)
14:52:03.5438453  System.Net.Security.HandshakeStop(protocol: 3072)
14:52:03.5440188  System.Net.Http.ConnectionEstablished(versionMajor: 1, versionMinor: 1, connectionId: 1, scheme: https, host: cache4-hkg1.steamcontent.com, port: 443, remoteAddress: 103.28.54.132)
14:52:03.5441639  System.Net.Http.RequestLeftQueue(timeOnQueueMilliseconds: 92.0418, versionMajor: 1, versionMinor: 1)
14:52:03.5442597  System.Net.Http.RequestHeadersStart(connectionId: 1)
14:52:03.5443555  System.Net.Http.RequestHeadersStop()
14:52:03.5809510  System.Net.Http.ResponseHeadersStart()
14:52:03.5810624  System.Net.Http.ResponseHeadersStop(statusCode: 401)
14:52:03.5811664  System.Net.Http.RequestStop(statusCode: 401)
[CDN] Failed to complete web request to https://cache4-hkg1.steamcontent.com/depot/440312/manifest/5480926750400450243/5: Response status code does not indicate success: 401 (Unauthorized).
Encountered 401 for depot manifest 440312 5480926750400450243. Aborting.

Unable to download manifest 5480926750400450243 for depot 440312
App 440310 was not completely downloaded.
The operation was canceled.
[Steam3Session] Disconnected: bIsConnectionRecovery = False, UserInitiated = True, bExpectingDisconnectRemote = False
Disconnected from Steam
```

### Additional Information

The game I tried to use this on is [Project Arrhythmia](https://steamdb.info/app/440310/), specifically the latest manifest in its `alpha` branch. It was working last week, but now it breaks.

I also notice that #521 reports a similar issue, but I think mine might be a different case because I am not in China, and my friend who is in Russia also encounters the same issue, at around the same time, on the same game.

We haven't tested it on any other game.

The official Steam client works fine, which makes it really odd.

**Comment by glgen** [comment]
I successfully downloaded a previous version of another game from the beta branch using the steam download_depot console command last night, but a few hours later the same command returned access denied and still not working. I think it may be related to this problem.

I just tried DepotDownloader, got 401. Passed the name of the beta branch as an -beta argument, and it starts downloading. Looks like steam now checks beta branch.
